### ISSUE TITLE
Make incorrect code examples accessible

### ISSUE THREAD

**Issue by TimJentzsch** [bevyengine/bevy-website#418]
Code snippets that are incorrect, i.e. that don't compile or give other errors, are currently styled with a red border and background:
![Example of an incorrect code example that shows up in red on the website](https://user-images.githubusercontent.com/13908946/181997237-42418a41-ce07-4d5a-a300-e08057d5c983.png)

Unfortunately, this is not accessible. People with color blindness or who are dependent on screen readers might miss that this will not compile.

If possible, I suggest adding some styled text like "This will not compile" or something similar at the top of the box, or an other indicator that can be picked up by screen readers.

**Comment by Nilirad** [comment]
I think a border should be enough. Alternatively, we can have accessibility options, maybe in the footer, with options to adjust the style according to the needs of the person.

**Comment by TimJentzsch** [comment]
In my opinion accessibility should be the default and not hidden behind options. Especially because it's harder to find those options if you already have visual impairments.
